WebDriverAgentMac/WebDriverAgentLib/Vendor/CocoaAsyncSocket/GCDAsyncSocket.h

```diff
@@ -123,8 +123,8 @@ typedef NS_ERROR_ENUM(GCDAsyncSocketErrorDomain, GCDAsyncSocketError) {
  * By default, both IPv4 and IPv6 are enabled.
```

Comment:
Please refrain from changing vendor sources unless really necessary. These are hardcopies of open-source libs that we don't really want to modify